.github/workflows/prt_labels.yml

```diff
@@ -0,0 +1,44 @@
+name: Remove the PRT label, for the new commit
+
+on:
+  pull_request:
+    types: ["opened", "synchronize"]
```

Comment:
@omkarkhatavkar only `synchronize` should be good enough because anyways we dont have any PRT labels on new PR.